### PR TITLE
PICARD-1527: fix dialog default sizes

### PR DESCRIPTION
**Pull request by phw**
# Summary
Fix dialog default sizes
    

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem
- Dialogs defined in UI files had two different places where a default size is defined (in UI file and code). Removed definition from code and adapted UI files.
- The default size of the options dialog is too small to accommodate the content. It is indeed one of our smallest dialogs, despite its demanding content. Actually the .ui file was once set to a sensible default, this is used again now.

* JIRA ticket (_optional_): [PICARD-1527](https://tickets.metabrainz.org/browse/PICARD-1527)

# Solution

# Action

### CODE REVIEW

File: picard/ui/ui_options_interface_theme.py [hunk 1]
```diff
@@ -1,50 +0,0 @@
-# -*- coding: utf-8 -*-
-
-# Automatically generated - don't edit.
-# Use `python setup.py build_ui` to update it.
-
-from PyQt5 import QtCore, QtGui, QtWidgets
-
-
```
Comment (zas) [inline picard/ui/ui_options_interface_theme.py]: Why is this file removed?

Comment (phw) [reply picard/ui/ui_options_interface_theme.py]: It's a leftover from when you renamed it to "colors" :)

Comment (zas) [reply picard/ui/ui_options_interface_theme.py]: Oh well... ;)
